template-compose/app/src/main/java/co/nimblehq/template/compose/ui/ErrorMapping.kt

```diff
@@ -10,3 +11,6 @@ fun Throwable.userReadableMessage(context: Context): String {
         else -> message
     } ?: context.getString(R.string.error_generic)
 }
+
+fun Throwable.showToast(context: Context) =
+    Toast.makeText(context, userReadableMessage(context), Toast.LENGTH_SHORT).show()
```

Comment:
I think we might need a better naming for this file as with this new extension fun, it is no longer used for `ErrorMapping` only => `ThrowableExtension.kt` maybe? 🤔

Reply:
@minhnimble This `ErrorMapping` is not a common thing like other extensions. It's specific for the error showing of a project, and might not be used in other projects because we could switch between custom messages from different `Throwable`s.

![image](https://user-images.githubusercontent.com/16315358/231940433-b5678dd6-3fc4-4955-a163-f2657b7961f0.png)

So I don't think we should rename it to `ThrowableExtension` or move it to the common extension package. This looks like a project component instead of a common code to be reused 🤔

Reply:
well-noted @luongvo it is a good point to move the `showToast` to `ContextExt` too, let's go 🚀